spec/requests/api/v1/sessions/destroy_spec.rb

```diff
@@ -3,15 +3,24 @@
 describe 'DELETE api/v1/users/sign_out', type: :request do
   let(:user) { create(:user) }
 
-  it 'returns a successful response' do
-    delete destroy_user_session_path, headers: auth_headers, as: :json
-    expect(response).to have_http_status(:success)
+  context 'with a valid token' do
+    it 'returns a successful response' do
+      delete destroy_user_session_path, headers: auth_headers, as: :json
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'decrements the amount of user tokens' do
+      headers = auth_headers
+      expect do
+        delete destroy_user_session_path, headers: headers, as: :json
```

Comment:
Why not headers: auth_headers?

Reply:
Because `auth_headers` adds the new headers to the user, so if the expect block measures the tokens changing they will change by 0